### PR TITLE
i18n(es): update image.md

### PR DESCRIPTION
**Pull request by glopzel**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- Added caution note regarding alt text

### CODE REVIEW

File: src/pages/es/guides/images.md [hunk 1]
```diff
@@ -118,6 +118,12 @@ Después de [instalar la integración](/es/guides/integrations-guide/image/#inst
 Los componentes Astro `<Image />` y `<Picture />` no pueden ser usados con imágenes en la carpeta `public/`. Usa sintaxis estándar de HTML o Markdown en lugar de estos componentes.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/images.md:118]:
```suggestion
Los componentes de Astro `<Image />` y `<Picture />` no pueden ser usados con imágenes dentro de la carpeta `public/`. Usa la sintaxis estándar de HTML o Markdown en lugar de estos componentes.
```

File: src/pages/es/guides/images.md [hunk 2]
```diff
@@ -118,6 +118,12 @@ Después de [instalar la integración](/es/guides/integrations-guide/image/#inst
...
 :::
 
+:::caution
+Los componentes Astro `<Image />` y `<Picture />` requieren de un atributo `alt` que proporcione texto descriptivo para imágenes. Estos componentes lanzarán un error si no contienen un texto `alt`.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/images.md:122]:
```suggestion
Los componentes de Astro `<Image />` y `<Picture />` requieren de un atributo `alt` que proporcione un texto descriptivo para las imágenes. Estos componentes lanzarán un error si no contienen un texto `alt`.
```

**Review by kevinzunigacuellar — COMMENTED** [review]
thank you for this 🤩, I left a couple comments 🧐

**Review by fredoist — APPROVED** [review]
Great work, nothing else to add. ✔️

**Comment by Yan-Thomas** [comment]
Thanks @glopzel, @kevinzunigacuellar & @fredoist. We highly appreciate this constant effort 💜

**Pull request merged**